drizzle-orm/src/sql/expressions/conditions.ts

```diff
@@ -231,7 +231,7 @@ export const gte: BinaryOperator = (left: SQLWrapper, right: unknown): SQL => {
  * @see lte for greater-than-or-equal
  */
 export const lt: BinaryOperator = (left: SQLWrapper, right: unknown): SQL => {
-	return sql`${left} < ${bindIfParam(right, left)}`;
+  return sql`${left} < ${bindIfParam(right, left)}`;
```

Comment:
removing these space changes regardless if they are right or wrong will most likely help get this PR over the line for you

Reply:
I only make this comment because I really want to see this functionality merged in and when I first looked at the PR it seemed alot bigger then it actually is due to all the whitespace changes

Reply:
Thanks very much! I have just fixed it.